experiments/functional/blockdevice-reusability/add_blockdevice.yml

```diff
@@ -0,0 +1,13 @@
+---
+    - name: Getting the Block-device from each node
+      shell: >
+        kubectl get bdc -n {{ operator_ns }} -l openebs.io/storage-pool-claim={{ spc_pool_name }} -o json | jq '.items[] | select(.spec.blockDeviceNodeAttributes.hostName=="{{ outer_item }}") | .spec.blockDeviceName' | tr "\"" " " 
```

Comment:
add a  comment specifying what should be the value of outer_item where this util is called.